examples/middleware/package.json

```diff
@@ -12,12 +12,8 @@
     "server": "node dist/server/entry.mjs"
   },
   "dependencies": {
-    "astro": "workspace:*",
-    "svelte": "^3.48.0",
-    "@astrojs/node": "workspace:*",
-    "concurrently": "^7.2.1",
-    "unocss": "^0.15.6",
-    "vite-imagetools": "^4.0.4",
+    "@astrojs/node": "^5.1.2",
+    "astro": "^2.4.1",
     "html-minifier": "^4.0.0"
```

Comment:
@ematipico I changed the `workspace:*` to the usual semver identifier since these templates would be cloned as-is by `create-astro`, and it'll need the raw identifiers in order to install normally.

---

Also, not entirely related, but when running the middleware example (`pnpm dev`), following the login flow with username and password as `"a"`, I get in the terminal:

```
01:22:07 PM [middleware] Middleware doesn't work for endpoints that return a simple body. The middleware will be disabled for this page.
```

Seems like it's because we're returning nothing here that's causing the issue:

https://github.com/withastro/astro/blob/6a748958fcc54baa54d390d762a0252e76838811/examples/middleware/src/middleware.ts#L68

Reply:
We should probably remove the endpoint from the example... The warning was added later on, after a discussion with the team.